### ISSUE TITLE
Window title issue

### ISSUE THREAD

**Issue by BzfpWg9g** [louis77/tuner#49]
Installed on Manjaro LXDE (with gtk3-nocsd installed, so no client side decorations).

If I start tuner, the window title is the same as the executable: com.github.louis77.tuner. If the executable is renamed the window title changes accordingly. When a station is played the window title changes to "Unbenanntes Fenster" ("unnamed window"). This ist also how the application name shown in the desktop panel.

The relevant output of `xprop`:

```
_NET_WM_VISIBLE_ICON_NAME(UTF8_STRING) = "Unbenanntes Fenster"
_NET_WM_VISIBLE_NAME(UTF8_STRING) = "Unbenanntes Fenster"
```

Is it possible to set the properties to something more meaningful? It could be "Tuner" on start and the radio station or actual song title if something is played.

**Comment by louis77** [comment]
@BzfpWg9g Thank you for your bug report. Indeed I've also noticed that recently. I'm currently working on 1.4 and will address this issue.

**Comment by louis77** [comment]
That is now fixed with [version 1.4.0](https://github.com/louis77/tuner/releases/tag/1.4.0).
